File: src/test/java/com/github/qky666/selenidepom/test/java/mtp/BaseMtpMobileTest.java
package com.github.qky666.selenidepom.test.java.mtp;

import com.github.qky666.selenidepom.Config;
import org.junit.jupiter.api.BeforeAll;

public abstract class BaseMtpMobileTest extends BaseMtpTest {
    @BeforeAll
    static void setUpMobile() {
        System.setProperty("chromeoptions.mobileEmulation", "deviceName=Nexus 5");
        Config.INSTANCE.setPomVersion("mobile");
    }
}

File: src/test/kotlin/com/github/qky666/selenidepom/test/kotlin/mtp/BaseMtpMobileTest.kt
package com.github.qky666.selenidepom.test.kotlin.mtp

import com.github.qky666.selenidepom.Config
import org.junit.jupiter.api.BeforeAll

abstract class BaseMtpMobileTest : BaseMtpTest() {
    companion object {
        @BeforeAll
        @JvmStatic
        fun setUpMobile() {
            System.setProperty("chromeoptions.mobileEmulation", "deviceName=Nexus 5")
            Config.pomVersion = "mobile"
        }
    }
}
